.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "datalake/backend/processing/check_metadata_content/stac-spec"]
+	path = datalake/backend/processing/check_metadata_content/stac-spec
```

Comment:
* according Detailed Design, this job name (and directory name) should be `check-stac-metadata`.

Reply:
I'll have to change that to `check_stac_metadata` to be able to import from it from Python.